tests/tests.cpp

```diff
@@ -20,6 +22,13 @@ TEST(tests, issueBar){
 	bool returnValue = func_1;
 	EXPECT_EQ(true, returnValue);
 }
+
+///[issue]https://github.com/Team-10-But-Better/Craft/issues/64
+TEST(tests, issue64){
```

Comment:
Your tests only check if the values are correct at the beginning, but they do not verify that they are set to the correct value when you enter binocular view.